src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -156,6 +156,12 @@
   <data name="FilePathTooLongWarning" xml:space="preserve">
     <value>The file '{0}' path, name, or both are too long. Your package might not work without long file path support. Please shorten the file path or file name.</value>
   </data>
+  <data name="IconUrlAndIconWarning" xml:space="preserve">
+    <value>Both 'iconUrl' and 'icon' elements were detected. Consider removing the 'iconUrl' element since it will be deprecated. To learn more, go to https://aka.ms/deprecateIconUrl.</value>
```

Comment:
I don't think we should speak in future tense here. 

> Both 'iconUrl' and 'icon' elements were detected. Consider removing the 'iconUrl' element since it is deprecated.

Reply:
Good point. But, this message should not be showing here.

Reply:
This message is removed in the following PR: https://github.com/NuGet/NuGet.Client/pull/3006 ; consider it non-existent for now.